File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/UserPref_.java
package de.dimedis.mobileentry;

import android.content.Context;
import android.content.SharedPreferences;

public class UserPref_ implements UserPref {

    SharedPreferences pref;
    SharedPreferences.Editor editor;

    public UserPref_(Context context) {
        SharedPreferences pref = context.getSharedPreferences("UserPref", 0);
        this.pref = pref;
        this.editor = pref.edit();
    }

    @Override
    public String userSuid() {
        return null;
    }

    @Override
    public String userName() {
        return null;
    }

    @Override
    public String userFullName() {
        return null;
    }

    @Override
    public String userPrefs() {
        return null;
    }

    @Override
    public String usersBorder() {
        return null;
    }

    @Override
    public String listUserFunctions() {
        return null;
    }

    @Override
    public String userSession() {
        return null;
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/UserPref.kt
package de.dimedis
/**
 * Created by Softeq Development Corporation
 * http://www.softeq.com
 */
interface UserPref {
    //user_suid
    fun userSuid(): String?

    //user_name
    fun userName(): String?
    fun userFullName(): String?

    //Map<String, String>
    fun userPrefs(): String?

    //border
    fun usersBorder(): String?

    // List<String>
    fun listUserFunctions(): String?

    //user_session
    fun userSession(): String?
}